@navikt/core/react/src/accordion/Accordion.tsx

```diff
@@ -14,16 +15,55 @@ interface AccordionComponent
 }
 
 export interface AccordionProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * @default 'primary'
+   */
+  variant?: "default" | "neutral";
+  /**
+   * @default 'large'
+   */
+  headingsize?: "large" | "medium" | "small" | "xsmall";
```

Comment:
Vi bruker for det meste camelCase for propnavn 🐫 
```suggestion
  headingSize?: "large" | "medium" | "small" | "xsmall";
```

Reply:
Ja, jeg hadde først lyst til å bruke camelCase, men endret det pga det ga utslag fra React i console

<img width="643" alt="image" src="https://user-images.githubusercontent.com/819074/235634847-2ac4a807-0254-40fd-aeb2-a2b41f4c2fd8.png">

🤔 , camelcase gir mer mening, skal undersøke feilen litt mer

Reply:
kanskje det bare var storybook props faktisk

Reply:
ja tror det er denne i storybook `<Accordion {...props}>` + nItems i ``` args: {
    controlled: false,
    nItems: 2,
    variant: "default",
    headingsize: "medium",
    size: "medium",
  },```